version.go

```diff
@@ -2,4 +2,4 @@ package main
 
 // version of this package.
 // Update this value before release of new version of software.
-const version = "0.6.0"
+const version = "0.6.0-dev"
```

Comment:
Should this be `0.7.0-dev` since it is the dev of the next version?

Reply:
Good idea. Fixed.